pkg/controller/manager/manager_controller.go

```diff
@@ -444,9 +444,22 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 		return reconcile.Result{}, err
 	}
 
-	var tunnelSecret certificatemanagement.KeyPairInterface
-	var internalTrafficSecret certificatemanagement.KeyPairInterface
+	// We expect that the secret that holds the certificates for internal communication within the management
+	// K8S cluster is already created by the KubeControllers
+	internalTrafficSecret, err := certificateManager.GetKeyPair(r.client, render.ManagerInternalTLSSecretName, common.OperatorNamespace())
```

Comment:
I believe that we should now do GetOrCreate in the manager_controller and change the esubecontrollers to just GetCertificate() or GetKeyPair() depending on whether we need the private key for that controller.

Reply:
What if we install MCM without going through Enterprise ? Manager is the last component rendered.

Reply:
I'd like that to be in master, but not in v1.30 (where this will be picked to)